src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2085 @@
+      &order=eventDate:desc&pageSize=10&page=1
+
+Query for all events with a certain program and organisation unit for a
+specific tracked entity instance:
+
+    GET /api/tracker/events?orgUnit=DiszpKrYNg8
+      &program=eBAyeGv0exc&trackedEntityInstance=gfVxE3ALA9m
+
+Query for all events with a certain program and organisation unit older
+or equal to
+2014-02-03:
+
+    GET /api/tracker/events?orgUnit=DiszpKrYNg8&program=eBAyeGv0exc&endDate=2014-02-03
+
+Query for all events with a certain program stage, organisation unit and
+tracked entity instance in the year 2014:
+
+    GET /api/tracker/events?orgUnit=DiszpKrYNg8&program=eBAyeGv0exc
+      &trackedEntityInstance=gfVxE3ALA9m&occurredAfter=2014-01-01&occurredBefore=2014-12-31
+
+Retrieve events with specified Organisation unit and Program, and use `Attribute:Gq0oWTf2DtN` as
+identifier scheme
+
+    GET /api/tracker/events?orgUnit=DiszpKrYNg8&program=lxAQ7Zs9VYR&idScheme=Attribute:Gq0oWTf2DtN
+
+Retrieve events with specified Organisation unit and Program, and use UID as identifier scheme for
+orgUnits, Code as identifier scheme for Program stages, and _Attribute:Gq0oWTf2DtN_ as identifier
+scheme for the rest of the metadata with assigned attribute.
+
+    GET /api/tracker/events?orgUnit=DiszpKrYNg8&program=lxAQ7Zs9VYR&idScheme=Attribute:Gq0oWTf2DtN
+      &orgUnitIdScheme=UID&programStageIdScheme=Code
+
+##### Response format
+
+The `JSON` response can look like the following.
+
+Please note that field filtering (`fields=...`) support is planned but not yet implemented.
+
+```json
+{
+    "instances": [
+        {
+            "href": "https://play.dhis2.org/dev/api/tracker/events/rgWr86qs0sI",
+            "event": "rgWr86qs0sI",
+            "status": "ACTIVE",
+            "program": "kla3mAPgvCH",
+            "programStage": "aNLq9ZYoy9W",
+            "orgUnit": "DiszpKrYNg8",
+            "orgUnitName": "Ngelehun CHC",
+            "relationships": [],
+            "occurredAt": "2021-10-12T00:00:00.000",
+            "followup": false,
+            "deleted": false,
+            "createdAt": "2018-10-20T12:09:19.492",
+            "updatedAt": "2018-10-20T12:09:19.492",
+            "attributeOptionCombo": "amw2rQP6r6M",
+            "attributeCategoryOptions": "RkbOhHwiOgW",
+            "dataValues": [
+                {
+                    "createdAt": "2015-10-20T12:09:19.640",
+                    "updatedAt": "2015-10-20T12:09:19.640",
+                    "storedBy": "system",
+                    "providedElsewhere": false,
+                    "dataElement": "HyJL2Lt37jN",
+                    "value": "12"
+                },
+              ...
+            ],
+            "notes": []
+        }
+    ],
+    "page": 1,
+    "pageSize": 1
+}
+```
+#### Events single object endpoint `GET /api/tracker/events/{uid}`
+
+Purpose of this endpoint is to retrieve one event given its uid.
+
+##### Request syntax
+
+`GET /api/tracker/events/{uid}?fields={fields}`
```

Comment:
maybe we shouldn't mention fields query param here, since it's not implemented?

Reply:
👍